### ISSUE TITLE
Backspace and Delete actions result in an unexpected behavior

### ISSUE THREAD

**Issue by YoussefTaghlabi** [RobinHerbots/Inputmask#444]
Using the following configuration in v2.5.0:

```
$(<SELECTOR>).inputmask("decimal", { 
                radixPoint: ".", 
                groupSeparator: ",",
                groupSize: 3,
                digits: 2,
                autoGroup: true,
                allowPlus: false,
                allowMinus: true
            });
```
- Type in the following data into the input field: `1234567`
  ( This gets masked to: `1,234,567` )
- Place cursor on the far left. The cursor should be now right before `1`
- Press "Delete" OR "BackSpace"

Expected Result: `234,567`
Actual Result: `2`

Using the same configuration
- Type in the following data into the input field: `1234567`
  ( This gets masked to: `1,234,567` )
- Place cursor right after `1`, before the `,`
- Press "BackSpace"

Expected Result: `234,567`
Actual Result: `2`

Note that this behavior happens only when the option`groupSeparator` has a value.

**Comment by RobinHerbots** [comment]
@YoussefTaghlabi , 

Thx for reporting.

Fixed in version 2.5.1

**Comment by YoussefTaghlabi** [comment]
Works great.
Thanks!
